packager/export_test.go

```diff
@@ -0,0 +1,213 @@
+package packager
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/deislabs/cnab-go/bundle/loader"
+
+	"github.com/deislabs/duffle/pkg/imagestore"
+	"github.com/deislabs/duffle/pkg/imagestore/imagestoremocks"
```

Comment:
This should be imported from `deislabs/cnab-go`.

Reply:
We need to decide whether to copy across the two image store implementations too or whether to leave them and the constructor functions in duffle. I don't feel strongly - it might be simplest to copy the lot.

Reply:
I think they need to be copied over.

Reply:
This generates a cyclical import if left like this.